packages/mock-block-dock/src/debug-view/header.tsx

```diff
@@ -134,18 +134,19 @@ export const Header = ({ darkMode, setDarkMode }: Props) => {
         </Typography>
       </Link>
 
-      <Button onClick={() => setDarkMode((prev) => !prev)} sx={{ mr: 1 }}>
-        Dark Mode
-        {darkMode ? (
-          <OnSwitch sx={{ height: 20, width: 40, ml: 1.25 }} />
-        ) : (
-          <OffSwitch sx={{ height: 20, width: 40, ml: 1.25 }} />
-        )}
-      </Button>
+      {/* @todo restore when styling service is implemented */}
+      {/* <Button onClick={() => setDarkMode((prev) => !prev)} sx={{ mr: 1 }}> */}
+      {/*  Dark Mode */}
+      {/*  {darkMode ? ( */}
+      {/*    <OnSwitch sx={{ height: 20, width: 40, ml: 1.25 }} /> */}
+      {/*  ) : ( */}
+      {/*    <OffSwitch sx={{ height: 20, width: 40, ml: 1.25 }} /> */}
+      {/*  )} */}
+      {/* </Button> */}
 
       <Button onClick={() => setDebugMode(false)}>
         Preview Mode
-        <OnSwitch sx={{ height: 20, width: 40, ml: 1.25 }} />
```

Comment:
Likely unrelated to this PR, but for my understanding why does the block currently lose its font styling in "Preview Mode"?
<img width="1003" alt="image" src="https://user-images.githubusercontent.com/42802102/190978312-0bdbebd6-f3d2-4029-a475-5b1c0ce5a6ab.png">
<img width="1002" alt="image" src="https://user-images.githubusercontent.com/42802102/190978329-e13cf39c-4fec-419d-ad62-176bb4607e91.png">

Reply:
No MUI components are added in this view, so I'm assuming there is some styling / theming data that is also not loadd

Reply:
Ah that makes sense, thanks for explaining